distribution/openrct2.d.ts

```diff
@@ -1140,24 +1318,47 @@ declare global {
 
     interface RideSetSettingArgs extends GameActionArgs {
         ride: number;
-        setting: number; // see RideSetSetting in openrct2/actions/RideSetSettingAction.h
+        /** @see RideSetSetting in
+         * [src/openrct2/actions/RideSetSettingAction.h](../src/openrct2/actions/RideSetSettingAction.h)
+         */
+        setting: number;
         value: number;
     }
 
     interface RideSetStatusArgs extends GameActionArgs {
         ride: number;
-        status: number; // 0: closed, 1: open, 2: testing, 3: simulating
+        /**
+         * - `0`: Closed
+         * - `1`: Open
+         * - `2`: Testing
+         * - `3`: Simulating
+         */
+        status: number;
     }
 
     interface RideSetVehicleArgs extends GameActionArgs {
         ride: number;
-        type: number; // 0: number of trains, 1: number of cars per train, 2: ride entry
-        value: number; // number value or sub type
-        colour: number; // only used if type is ride entry
+        /**
+         * - `0`: Number of trains
+         * - `1`: Number of cars per train
+         * - `2`: Ride entry
+         */
+        type: number;
+        /**
+         * Number value or subtype of {@link RideSetVehicleArgs.type}
+         */
+        value: number;
+        /**
+         * Only used if {@link RideSetVehicleArgs.type} === 2 (ride entry)
+         */
+        colour: number;
     }
 
     interface ScenarioSetSettingArgs extends GameActionArgs {
-        setting: number; // see ScenarioSetSetting in openrct2/actions/ScenarioSetSettingAction.h
+        /** @see ScenarioSetSetting in
+         * [src/openrct2/actions/ScenarioSetSettingAction.h](../src/openrct2/actions/ScenarioSetSettingAction.h)
```

Comment:
Nitpicky, but seems more consistent with the rest:

```suggestion
        /** 
         * @see ScenarioSetSetting in [src/openrct2/actions/ScenarioSetSettingAction.h](../src/openrct2/actions/ScenarioSetSettingAction.h)
```